docs/user/alerting/defining-alerts.asciidoc

```diff
@@ -94,6 +94,10 @@ Some cases exist where the variable values will be "escaped", when used in a con
 
 Mustache also supports "triple braces" of the form `{{{variable name}}}`, which indicates no escaping should be done at all.  Care should be used when using this form, as it could end up rendering the variable content in such a way as to make the resulting parameter invalid or formatted incorrectly.
 
+Each alert type defines additional variables as properties of the variable `context`.  For example, if an alert type defines a variable `value`, it can be used in an action parameter as `{{context.value}}`.  
+
+For diagnostic or exploratory purposes, action variables whose values are themselves objects, such as `context`, can be referenced directly as variables.  The resulting value will be a JSON representation of the object.  For example, if an action parameter includes `{{context}}` in an action parameter, it will expand to the JSON representation of all the variables and values provided by the alert type.
```

Comment:
```suggestion
For diagnostic or exploratory purposes, action variables whose values are themselves objects, such as `context`, can be referenced directly as variables.  The resulting value will be a JSON representation of the object.  For example, if an action parameter includes `{{context}}`, it will expand to the JSON representation of all the variables and values provided by the alert type.
```